README.md

```diff
@@ -1,2 +1,69 @@
 # Fall2019CourseProjectTDA
 The RAIK383H course project for TD Ameritrade during Fall semester 2019.
+
+# GitHub Student
+GitHub offers unlimited private repositories to students. Your code is required to be in a private repository to adhere to UNL's academic integrity policy.
+
+A .edu email address is required to apply to GitHub Student. Navigate to [settings](https://github.com/settings/emails) and add your huskers.unl.edu address to your list of emails.
+
+Navigate to [GitHub Student](https://education.github.com/pack), log in with your GitHub account, and click "Get your Pack". Follow the instructions, verifying your identiy as necessary. It may take multiple days for GitHub to approve your request, so apply early.
+* Note: due to a policy change by GitHub involving huskers.unl.edu addresses, your huskers.unl.edu email may not sufficiently verify your identity as a student. If you have an option to submit additional identification, you may submit a picture of your NCard or download an Unofficial Transcript from MyRed (which shows your in progress classes for this semester). GitHub support said that an NCard may not be sufficient, but upperclassmen have used it successfully.
+
+# Cloning the Repo
+Only 1 person needs to clone the repo per team. Once this section is done, the remaining members of the team will clone the new repository.
+
+## Option 1
```

Comment:
This is really cool!

Reply:
Yah ikr, I just learned about this while setting up this repo